nav2_experimental_rl/README.md

```diff
@@ -0,0 +1,60 @@
+# Overview
+The ROS 2 Experimental Reinfocement Learning (RL) package is a package that enables to train and control a robot with RL techniques.  This package is utilizing Gazebo simulator and ROS 2 to train TurtleBot 3 robot. 
```

Comment:
"This package is utilizing **the** Gazebo simulator and ROS 2 to trans **a** TurtleBot 3 robot."